playground/src/components/DropDown.tsx

```diff
@@ -18,11 +18,13 @@ type DropDownProps = {
   onExampleValueChange?: (newType: string) => void;
 };
 
-export default function DropDown({ onExampleValueChange }: DropDownProps): JSX.Element {
+export default function DropDown({
```

Comment:
This is only made prettier using `npx prettier --write .`. I wasn't able to undo that step. I can manually remove those files from the PR if you want.